system-variables.md

```diff
@@ -1338,6 +1346,14 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 - Default value: `ON`
 - This variable is used to control whether to enable the index merge feature.
 
+### tidb_enable_index_merge_join
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Type: Boolean
+- Default value: `false`
+- Specifies whether to enable the `IndexMergeJoin` operator. This variable is used only for the internal operation of TiDB. It is **NOT recommended** to adjust it. Otherwise, data correctness might be affected.
```

Comment:
```suggestion
- Specifies whether to enable the `IndexMergeJoin` operator.
- This variable is used only for the internal operation of TiDB. It is **NOT recommended** to adjust it. Otherwise, data correctness might be affected.
```